### DISCUSSION TITLE
中央音樂遊戲研究社第一次社課

### DISCUSSION THREAD

**Discussion by v0ouo** [1101-GS4719-C/Python-Community-GS4719-C#1789] [General]
借我推廣一下ouo

首先，我們社團是經過1年多才創立的，DC當初從小小的6個人，到現在已經有10倍以上的人了，這一路走來真的很不容易，我認為這個圈子已經夠小了，能加進來跟有共同興趣的人一起交流有什麼不好的呢，所以加進來一起交流吧!

以下是宣傳的部分:
你也喜歡打音遊或是對音遊有興趣嗎？歡迎來第一堂社課喔！
時間：12/19 19:00~22:30
地點：游藝館三樓 游會

對
沒錯
中央終於有音遊社了
如果你跟我們一樣在來到這間學校時第一個反應是「蛤？中央竟然沒有音遊社？」那麼沒事，因為我們來了
中央音遊於2023年10月底正式成立，可惜的是遇到了隨之而來的期中考（還有社長的怠惰）使得第一次的社團活動一直拖到了12月
本次社課的主題是maimai介紹，講者為主坑maimai的社長本人。從入門須知到進階知識，讓不論是想入坑卻不知道如何踏出第一步的你，又或者已經是資深玩家的你，皆能在這堂課中收穫滿滿
此外，本次社課也將進行入社說明，歡迎對音樂遊戲有興趣的大家一同參與
不論你是移動端玩家或是街機玩家，本社皆有經驗豐富的社員可進行技術交流，未來的社課內容也十分豐富，從技術教學到譜面創作甚至是手台製作皆有囊括，未來也計畫舉辦交流賽等各類活動，心動的同時就趕快動手填寫入社表單吧！[https://forms.gle/RkzKLztqBsHrR5eV6](https://l.facebook.com/l.php?u=https%3A%2F%2Fforms.gle%2FRkzKLztqBsHrR5eV6%3Ffbclid%3DIwAR1I_7oayeirVRCW8pz6oG6zXMZ_Z2HYk6o2qsRMk24KyOW5Py22JgbkiOA&h=AT0IvewMzN-fj2xv5XH5-ojkMilOGc9i4uahDv8hptdtKyF0Q2NSqwvG-qw-j6si_xEJ1Z2GqHhdwFik0lXqXrMl7NIQz_EJZ62_a9V2u_MKfe4fUf3e_l-r8grUUmiv4IUs&__tn__=-UK-y-R&c[0]=AT1Ezo9tLl6cq9rvKK6xwtaICuyngrFOUnunS7JBo9rVgQE8PptDCZLx2KXcYPcgJOsktMkDpn0KCCNSnFNeb196jYQ8GMA1DgX4kfW8Vogr86RHej9yDqmL9TSrDPKudL8Lm4P5b1X_aXsgg2Q42ieiNc4alrDiS5Z_dEwAUgrPz-XddF_a2gUVtUtrhuDq01EfdFP0KnTfT4TLG-KlBkNx2X2-77OjSW45hDHFm8karUE)
另外也歡迎加入我們的dc群獲得更多消息：[https://discord.gg/V4sVHWNG](https://l.facebook.com/l.php?u=https%3A%2F%2Fdiscord.gg%2FV4sVHWNG%3Ffbclid%3DIwAR18-YsX3ZOH7vKp-BVswJqpAsh1Y1BcyNFf2kH1jOO0VZXOIL5PDTqTsGI&h=AT3cDrHE_0a2cZUWLurQEJAg-2jSRkeI9DgDaz6PNLQfoLiN5nS5EQvvMYP6juXQCHjpeKo7bHCGvE0pmc71_IBBqKlRQe_ZxyUkmN-fRx8EuvQ-ShsbTwXqNEbgvyYm0DOR&__tn__=-UK-y-R&c[0]=AT1Ezo9tLl6cq9rvKK6xwtaICuyngrFOUnunS7JBo9rVgQE8PptDCZLx2KXcYPcgJOsktMkDpn0KCCNSnFNeb196jYQ8GMA1DgX4kfW8Vogr86RHej9yDqmL9TSrDPKudL8Lm4P5b1X_aXsgg2Q42ieiNc4alrDiS5Z_dEwAUgrPz-XddF_a2gUVtUtrhuDq01EfdFP0KnTfT4TLG-KlBkNx2X2-77OjSW45hDHFm8karUE)

**Comment by mrweixd** [comment]
副社長好帥

**Comment by eric202501** [comment]
我也好想加入中央音樂遊戲研究社，可惜我的室友都不揪我去，我只能待在宿舍啃二元樹

**Comment by woohsin** [comment]
從高中畢業那年被我地推坑開始打mai，但我太矮了，打不到外圈最上面那兩個，從開始到現在都只打過內圈，而且他每次改版去刷卡都會發現R值少了好多，心在淌血阿，畢竟那是我用錢堆出來的...

**Reply by 20050201Shawn** [reply]
覺得打mai最佳身高是170左右(我沒有)，隔一陣子去打多了好多新歌，刷新歌比較好洗R值(x

**Reply by woohsin** [reply]
打mai對身高也太苛求了吧 ;) 我之前會看到長太高的人微蹲(幾乎快90度...)，想想就覺得他們好累
我也是都直接開新歌打到底哈哈，一更新就開始刷
